internal/controller/vrg_volrep.go

```diff
@@ -420,6 +453,17 @@ func (v *VRGInstance) protectPVC(pvc *corev1.PersistentVolumeClaim, log logr.Log
 
 			return requeue
 		}
+
+		if rmnutil.IsCGEnabled(v.instance.GetAnnotations()) {
```

Comment:
We already labelled the PVC with the CG label, why add it again here? Is it because the PVC that we have is a stale copy?